### ISSUE TITLE
Introduction page to the course

### ISSUE THREAD

**Issue by szabgab** [kantord/LibreLingo#1842]
I think it would be nice to be able to write some introduction text for each course in an md file (e.g. `course.md` next to `course.yaml`) that would be displayed when the user first reaches the course. Then once the user saw that page once it could be hidden behind a link.
So the user can still display it but it is not the default page any more.

I would put there some explanation about the course, how to contribute, how to reach the author(s) etc. This could be written in the source language.

**Comment by frostedkitsune** [comment]
>Home page -> Select a course -> Choice start learning or log in to existing account -> Complete first skill -> Choice register or continue without account -> Course introduction page

I would rather suggest:
Home page -> librelingo pwa -> install or skip -> cousre list -> select a course -> Choice start learning or log in to existing account or create an acount -> Complete first skill -> Choice register or continue without account -> Course introduction page

It would be great to store progess even if an user isn't continuing with an account but installed the app so that, if they decide to sign up/sign in later, the progess could be restored

**Comment by frostedkitsune** [comment]
anyway, is account creation and authentication implemented yet?

**Comment by kantord** [comment]
> anyway, is account creation and authentication implemented yet?

yes it is. I'm confused because a lot of people ask this and I don't understand why. I thought it's in a pretty obvious place. Do you think there's a UX/design issue which makes people not notice where the logic screen is?

**Comment by kantord** [comment]
> Home page -> librelingo pwa -> install or skip

wouldn't this simply increase the churn rate? Me personally, even if I do install an actual app, let alone a PWA, it doesn't mean I'll use it at all 🤔 I haven't seen numbers for this though, so I'm just talking out of personal experience. But in summary I don't see any value in pushing people to install a PWA/app really early and hard. Unless it's for the purposes of spamming them with notifications. Actually we don't use notifications at all right now. Maybe this becomes important if we have an actually useful and legit reason to send notifications to people

**Comment by kantord** [comment]
> It would be great to store progess even if an user isn't continuing with an account but installed the app so that, if they decide to sign up/sign in later, the progess could be restored

this is already implemented even without having to install the PWA. All storage is offline first, stored in localStorage. Login is only used for backup to a database server and synchronizing between multiple devices

**Comment by frostedkitsune** [comment]
> > anyway, is account creation and authentication implemented yet?
> 
> yes it is. I'm confused because a lot of people ask this and I don't understand why. I thought it's in a pretty obvious place. Do you think there's a UX/design issue which makes people not notice where the logic screen is?

Yeah, now I see but never noticed before, I feel the design done wrong!

**Comment by frostedkitsune** [comment]
> Unless it's for the purposes of spamming them

Just prompting once isn't spamming, I think :)

**Comment by frostedkitsune** [comment]
> this is already implemented even without having to install the PWA......

Oh, great job!

**Comment by kantord** [comment]
> Just prompting once isn't spamming, I think :)

I can be convinced of this of course
